gradle/libs.versions.toml

```diff
@@ -41,4 +41,4 @@ benManesVersions = { id = "com.github.ben-manes.versions", version = "0.51.0" }
 binaryCompatibilityValidator = { id = "org.jetbrains.kotlinx.binary-compatibility-validator", version = "0.14.0" }
 
 [bundles]
-defaultTests = ["junit", "kotestAssertions"]
\ No newline at end of file
+defaultTests = ["junit", "kotestAssertions"]
```

Comment:
Is this intentional?

Reply:
No. I used GitHub UI and got this behavior. I suspect that UI adds another line separator or one more line separator.